### PR TITLE
add a version to the travis deploy, allow arcs-live to have a differe…

### PR DESCRIPTION
**Pull request by smalls**
…nt README

### CODE REVIEW

**Comment by smalls** [comment]
A further iteration on #825.

File: .travis.yml [hunk 1]
```diff
@@ -22,6 +22,9 @@ before_deploy:
   # Travis pages deploys skips artifacts from .gitignore. We need some of those, so
   # remove them from .gitignore.
   - cd ${TRAVIS_BUILD_DIR} && sed -i.sedbackup -e 's/^shell\/build$//g' .gitignore
```
Comment (dstockwell) [inline .travis.yml]: Didn't notice this earlier, but perhaps we could just remove the entire .gitignore file?

Comment (smalls) [reply .travis.yml]: That would also include node_nodules & bower_components. That'd make the deploy pretty large, but might help with your use of `index.debug.html`?

Actually, maybe too large:
```
> du -sh arcs
938M	arcs
> du -sh arcs/node_modules/
530M	arcs/node_modules/
```

!!!

Comment (smalls) [reply .travis.yml]: Is there a subset, or other files that'd be helpful? I could always clear out `node_modules` & `bower_components`.

Comment (dstockwell) [reply .travis.yml]: ahh! I forgot about node modules, seems like the approach you have is probably fine for now... I guess we can extend it as needed.

Comment (smalls) [reply .travis.yml]: 👍

**Pull request merged**